src/sentry/static/sentry/app/components/dropdownAutoComplete/menu.tsx

```diff
@@ -191,6 +192,7 @@ type Props = {
    */
   rootClassName?: string;
 
+  zIndex?: number;
```

Comment:
Please, is this really being used? I remember that it wasn't making any difference

Reply:
Looks like it might indeed not be used 👍